Website/web/src/layout/Header/AccountMenu.jsx

```diff
@@ -65,6 +80,11 @@ const AccountMenu = () => {
                     Logout
                 </MenuItem>
             </Menu>
+            <div>
```

Comment:
We might want to consider refactoring this so that it is grouped with the AccountCircleIcon to ensure that clicking on the user's name also displays the dropdown menu.